src/clj/rems/api/applications.clj

```diff
@@ -163,7 +156,7 @@
       (-> application
```

Comment:
The handler and other user permissions are also sensitive information that should be hidden

Reply:
The permissions are removed by calling `rems.workflow.permissions/cleanup` before it is returned from the public API. The API schema should prevent them slipping through accidentally.

The list of handlers in the workflow should probably be hidden from the applicant.

Reply:
Handlers are now hidden.